test/block/blockheader.js

```diff
@@ -52,8 +51,8 @@ describe('BlockHeader', function() {
 
   describe('#constructor', function() {
 
-    it('should set all the variables', function() {
-      var bh = new BlockHeader({
+    it('should set all the constiables', function() {
```

Comment:
constants instead of constiables?

Reply:
variables instead, I did intellij replace var by const, and it modified the wrong one 😅